File: smiex_api/smiex_api/Models/Paisso.cs
using System;
using System.Collections.Generic;

namespace smiex_api.Models;

public partial class Paisso
{
    public int Id { get; set; }

    public string? Nom { get; set; }

    public virtual ICollection<Ciutat> Ciutats { get; set; } = new List<Ciutat>();
}

File: simex_app/app/src/main/java/com/example/simex_app/data/models/Paisso.kt
package com.example.simex_app.data.models

data class Paisso(
    val id: Int,
    val nom: String?,

    // Relación con las Ciudades (Uno a Muchos)
    // Inicializamos con lista vacía para evitar errores de nulabilidad
    val ciutats: List<Ciutat> = emptyList()
)